openslides_backend/action/actions/chat_group/clear.py

```diff
@@ -0,0 +1,37 @@
+from ....models.models import ChatGroup
+from ....permissions.permissions import Permissions
+from ....shared.patterns import FullQualifiedId
+from ....shared.schema import required_id_schema
+from ...generics.update import UpdateAction
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData
+from ..chat_message.delete import ChatMessageDelete
+from .mixins import ChatEnabledMixin
+
+
+@register_action("chat_group.clear")
+class ChatGroupClear(ChatEnabledMixin, UpdateAction):
+    """
+    Action to clear a chat group.
+    """
+
+    model = ChatGroup()
+    schema = DefaultSchema(ChatGroup()).get_default_schema(
+        title="Schema for clear a chat group.",
+        additional_required_fields={"chat_group_id": required_id_schema},
```

Comment:
I would think the `chat_group_` prefix is implicitly given by the action name, so only `id` as the parameter name should suffice. (@GabrielInTheWorld)

Reply:
(then you can just use the normal update schema here)

Reply:
Done.